### ISSUE TITLE
Breaks Teamviewer website and application

### ISSUE THREAD

**Issue by gut5** [StevenBlack/hosts#762]
Tested with Teamviewer 13 in Windows, Teamviewer works fine in Windows with usual hosts file, but with this hosts file it won't connect to Teamviewer server and also the website won't load.

**Comment by dnmTX** [comment]
website works perfectly fine here on windows.Some domains are blocked but they're all nasty ones and don't affect the browsing:
tracking.g2crowd.com  
glancecdn.net
s.adroll.com

**Comment by kronflux** [comment]
Yeah I'm not experiencing this at all. I use TeamViewer pretty heavily for work. I can't say anything for the website, as I've only ever touched it to download the client initially, but the client is working perfectly.

**Comment by StevenBlack** [comment]
Hi @gut5 can you give us a clear example of how Teamviewer is broken for you?
